base.config.js

```diff
@@ -57,20 +59,24 @@ module.exports = {
             video: 'src',
             source: 'src'
           },
+          loaders: { ts: 'awesome-typescript-loader' }
         }
       },
       {
         test: /\.ts$/,
-        loader: 'ts-loader',
-        options: { experimentalWatchApi: true },
+        loader: 'awesome-typescript-loader',
+        options: { useCache: true, forceIsolatedModules: true, reportedFiles: ['app/**/*.ts'] },
```

Comment:
Why we need `isolatedModules` here?

Reply:
it's just mentioned in the `awesome-typescript-loader` docs as something that improves build speed, but if there's a reason to not have it would definitely be ok without it

Reply:
As I see from docs:
> Transpile each file as a separate module (similar to 'ts.transpileModule'). 
That means we're gonna lose cross-file type checking with this option.

Reply:
afaik this only effects the transpilation process and not the typechecking process, which runs in a separate thread